src/bpftrace.cpp

```diff
@@ -1445,6 +1467,15 @@ uint64_t BPFtrace::resolve_uname(const std::string &name, const std::string &pat
   return addr;
 }
 
+std::string BPFtrace::extract_func_symbols_from_path(const std::string &path)
+{
+  std::string call_str = std::string("objdump -tT ") + path +
```

Comment:
It's minor, but I'd have a comment above this that said something like "// XXX Shell out to objdump for now; in the future, replace with library calls.".

Just so we remember to one day explore using libelf or whatever for this.

Reply:
The function above has a TODO item for that very issue, so I copied it into this one.